great_expectations/datasource/pandas_datasource.py

```diff
@@ -282,10 +282,13 @@ def guess_reader_method_from_path(path):
                 "reader_method": "read_csv",
                 "reader_options": {"compression": "gzip"},
             }
+        elif path.endswith(".sas7bdat") or path.endswith(".xpt"):
+            return {"reader_method": "read_sas"}
 
-        raise BatchKwargsError(
```

Comment:
Can you return this raised exception to it's original depth?

Reply:
@AFineDayFor  - Can you point me to an example of what a V3 API test should look like? Thanks.

Reply:
Hey and howdy @andyjessen, you can refer to [this](https://github.com/great-expectations/great_expectations/blob/develop/tests/execution_engine/test_pandas_execution_engine.py#L208-L217) as an example.

Reply:
Thanks. I'm still not understanding.

If I modify the  test_reader_fn  to include the read_sas reader method, would that work? Also, does it need a actual file? I looked for the "myfile.xlsx" to see how this function was called, and couldn't find it.

Reply:
The function in the test verifies that if you declare the file-type, the correct reader method will be returned; it does not require a file.

Reply:
Thanks @AFineDayFor. I modified the existing test in 37c741c. How do I modify the documentation?